features/steps/steps.py

```diff
@@ -139,6 +139,13 @@ def then_stdout_matches_regexp(context):
     assert_that(context.process.stdout.strip(), matches_regexp(context.text))
 
 
+@then("stdout does not match regexp")
```

Comment:
good add once your branch lands I'll rebase mine on master and use this step too